service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/UploadC2DocumentsAboutToSubmitControllerTest.java

```diff
@@ -43,9 +40,6 @@
     private static final Long CASE_ID = 12345L;
     private static final DocumentReference document = testDocumentReference();
 
-    @MockBean
-    private IdamClient idamClient;
-
     @MockBean
     private RequestData requestData;
```

Comment:
Do you wanna do the same for request data?

Reply:
Also you can remove time from this one

Reply:
good point. Yes i would like to have this for RequestData as well

Reply:
done